python/cuml/dask/common/part_utils.py

```diff
@@ -128,8 +129,9 @@ def _extract_partitions(dask_obj, client=None):
     client = default_client() if client is None else client
 
     # dask.dataframe or dask.array
```

Comment:
Probably want to update this comment as well.

Reply:
Good catch!